### PR TITLE
Add Shin Megami Tensei: Devil Survivor Overclocked

### PR DESCRIPTION
**Pull request by dualslash**
Adds Shin Megami Tensei: Devil Survivor Overclocked.

*Automatically generated by PR builder*

### CODE REVIEW

File: shin-megami-tensei-devil-survivor-overclocked/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+release_date = "2013-03-29"
+
+[[testcases]]
+title = "000400000009C000"
+compatibility = "1"
+date = "2017-07-14"
+version = "HEAD-669757a"
+cpu = "Intel Core i5-3570K @ 4.2Ghz"
+gpu = "GeForce GTX 1060"
+os = "Windows 7 17601"
```
Comment (Hexagon12) [inline shin-megami-tensei-devil-survivor-overclocked/game.dat]: _Windows 7 SP1_

File: shin-megami-tensei-devil-survivor-overclocked/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
+title = "Shin Megami Tensei: Devil Survivor Overclocked"
+description = "Tokyo is in full lockdown, the government's swift, coldblooded response to a surprise demon invasion. While you and your friends must solve the nature of the interdimensional rift through which the horrific creatures are entering your world, saving the world is not your highest priority; staying alive, on the other hand, is."
+needs_system_files = false
+needs_shared_font = false
+
```
Comment (Hexagon12) [inline shin-megami-tensei-devil-survivor-overclocked/game.dat]: There is a issue surrounding this game on our GitHub, could you add it here?

File: shin-megami-tensei-devil-survivor-overclocked/game.dat (hunk 1 above)
Comment (j-selby) [inline shin-megami-tensei-devil-survivor-overclocked/game.dat]: Hexagon: I recommended specific build versions as they are consistent between window versions.

Comment (Hexagon12) [inline shin-megami-tensei-devil-survivor-overclocked/game.dat]: Then it should be _Windows 7 7601_, not **1**7601.

**Comment by Schplee** [comment]
2 weeks have passed and no sign of the author complying with our changes. Also missing screenshots. Closing PR.

**Pull request closed without merging**